terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -268,10 +268,13 @@ resource "kubernetes_cron_job" "workflow_manager" {
               name  = "workflow-manager"
               image = "${var.container_registry}/${var.workflow_manager_image}:${var.workflow_manager_version}"
               args = [
+                "--is-first", var.is_first ? "true" : "false",
                 "--k8s-namespace", var.kubernetes_namespace,
                 "--k8s-service-account", kubernetes_service_account.workflow_manager.metadata[0].name,
                 "--ingestor-input", "s3://${var.ingestion_bucket}",
                 "--ingestor-identity", var.ingestion_bucket_role,
+                "--own-validation-bucket", "gs://${var.own_validation_bucket}",
+                "--peer-validation-bucket", "s3://${var.peer_validation_bucket}",
```

Comment:
Not for the `gs://` bucket, as it's in the same cloud as us and we set it up so the workflow manager's GCP service account can use it, so nothing special is required. We could add a `peer-validation-identity` though it'd have the same value as `ingestor-identity`.

Reply:
I'd like to have a `--peer-validation-identity` even though it's the same as `ingestor-identity`. It documents to someone reading the terraform what identity is used to access that bucket, since we've established the pattern that `--foo-{input,output}` flags come in pairs with `--foo-identity` flags.